From e597ffe258d510defbd4eadef2b0b4d07b903b47 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Mon, 14 May 2018 15:22:14 -0700
Subject: [PATCH] TRITON-385 cloudapi has a vestigial logadm.conf entry for
 /var/log/cloudapi.log Reviewed by: Richard Kiene <richard.kiene@joyent.com>
 Approved by: Richard Kiene <richard.kiene@joyent.com>

---
 boot/setup.sh | 1 -
 1 file changed, 1 deletion(-)

diff --git a/boot/setup.sh b/boot/setup.sh
index c9e2eab..0775d3f 100755
--- a/boot/setup.sh
+++ b/boot/setup.sh
@@ -160,7 +160,6 @@ sdc_log_rotation_add cloudapi-8081 /var/svc/log/*cloudapi-8081.log 1g
 sdc_log_rotation_add cloudapi-8082 /var/svc/log/*cloudapi-8082.log 1g
 sdc_log_rotation_add cloudapi-8083 /var/svc/log/*cloudapi-8083.log 1g
 sdc_log_rotation_add cloudapi-8084 /var/svc/log/*cloudapi-8084.log 1g
-sdc_log_rotation_add cloudapi /var/log/cloudapi.log 1g
 sdc_log_rotation_setup_end
 
 # All done, run boilerplate end-of-setup
-- 
2.21.0

